roles/kubernetes/preinstall/tasks/main.yml

```diff
@@ -1,4 +1,10 @@
 ---
+- name: 'GIT | Install script for collecting git info'
+  template:
+    src: "{{ role_path }}/gen-gitinfos.sh"
```

Comment:
I don't like this path so much. Let's move gen-getinfos.sh to roles/kubernetes/preinstall/files/gen-getinfos.sh and then write this line as src: gen-getinfos.sh